.github/workflows/auto-build.yml

```diff
@@ -0,0 +1,44 @@
+name: Auto build image
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: ${{ github.event.comment.body == '/make build v1.20.14 ADM64' }}
```

Comment:
'v1.20.14' and 'ADM64' are variables that need to be specified by passing in parameters.

Reply:
yes. I'll run through the logic first, and then add variables